packages/blade/src/components/Amount/Amount.tsx

```diff
@@ -120,6 +123,7 @@ const AmountValue = ({
           fontSize={amountFontSizes[size]}
           fontWeight={valueForWeight}
           color={amountValueColor}
+          as={isReactNative ? undefined : 'span'}
```

Comment:
`AmountWrapper` is rendering a `div`, should it be `p` tag instead?

Reply:
We need that for alignment in native, have left a comment.

Reply:
I meant for web it's rendering a `div` container

Reply:
<img width="245" alt="image" src="https://user-images.githubusercontent.com/63398163/229030849-3f187619-d24e-4de0-9b82-0141b508576c.png">

Only the external most container is a div.

Reply:
AmountWrapper is rendering a fragment.